### PR TITLE
Updated original files with working VS projects

### PR DESCRIPTION
**Pull request by soradotwav**
- Rewrote and updated parts of all original v_backup sample files to work with the newest release of C++
- Created VS projects for each of the samples
- Rewrote each sample to be fully runnable and functional (to the extend commented on within the files themselves)

### CODE REVIEW

File: samples/features/sqlvdi/mprocess/mprocess.cpp [hunk 1]
```diff
@@ -44,216 +44,220 @@ All Rights Reserved.
...
-    IClientVirtualDeviceSet *vds,
-    HANDLE  hSQLProcess,  // handle to process dealing with the SQL
-    int     nStreams,     // number of i/o streams
-    char*   pgmName);    // the name of this program
+int startSecondaries(
+	IClientVirtualDeviceSet2*	vds,
+	HANDLE						hSQLProcess,	// handle to process dealing with the SQL
+	int							nStreams,		// number of i/o streams
+	char*						pgmName			// the name of this program
+);
```
Comment (MichaelEbert) [inline samples/features/sqlvdi/mprocess/mprocess.cpp:66]: spacing

File: samples/features/sqlvdi/mprocess/mprocess.cpp [hunk 2]
```diff
@@ -263,50 +267,54 @@ int main(int argc, char *argv[])
...
-    // If you want to use Windows NT Authentication, please do not use the -U or -P options.
-    sprintf (cmd, 
-        "osql -E -b -Q\"%s DATABASE PUBS %s VIRTUAL_DEVICE='%ls'",
-        (doBackup) ? "BACKUP" : "RESTORE",
-        (doBackup) ? "TO"     : "FROM",
+	WCHAR					cmd[5000];
+	WCHAR					extend[100];
+	PROCESS_INFORMATION     pi;
+	STARTUPINFO             si;
+	int                     ix;
```
Comment (MichaelEbert) [inline samples/features/sqlvdi/mprocess/mprocess.cpp:276]: spacing

**Review by MichaelEbert — CHANGES_REQUESTED** [review]
It looks like the original files were created with spaces instead of tabs. Can we use spaces as well?

**Review by MichaelEbert — APPROVED** [review]
Looks good.

**Review by swells — APPROVED** [review]

**Pull request merged**